db/nft.go

```diff
@@ -349,13 +349,21 @@ func GetNftEvents(conn *pgxpool.Conn, q QueryEventsRequest, p PageRequest) (Quer
 	return res, nil
 }
 
+func getSourceTable(priceBy string) string {
+	switch priceBy {
+	case "class":
+		return "c"
+	case "nft":
+	default:
+		return "n"
+	}
+	return "n"
+}
```

Comment:
Should check the value is `"class" | "nft"` at somewhere (probably REST handler) and return error for invalid field instead of silently falling back to default, otherwise it could make some mysterious bugs (e.g. `price_by=classes` gets prices by NFT)